srcOA3/mixins/response/absoluteTest.yml.njk

```diff
@@ -0,0 +1,6 @@
+application/json:
+  schema:
+    type: object
+    properties:
+      included:
+        $ref: {{ mixinDirectory }}/referredFromAbsolute.yml.njk
```

Comment:
Side note: I did try using absolute paths inside a mixin, but ran into a flavour of the problem that `{{ mixinDirectory }}` tries to solve. In that the absolute path is calculated relative to the mixin file, and then the mixin is rendered into a different location.

At the moment I'm leaving that as a known issue, unless anyone thinks it's critical to resolve.